Sources/Impl/PinLayoutImpl+Warning.swift

```diff
@@ -152,7 +152,8 @@ extension PinLayoutImpl {
             #endif
         }
 
-        displayText += ", Tag: \(view.tag))\n"
+        // Removed, layoutable tag should probably not be exposed
+        //displayText += ", Tag: \(view.tag))\n"
```

Comment:
Maybe the Layoutable should expose a property `warningContext`, that would expose a String returning this block content: 
``` 
var hierarchy: [String] = []
....
displayText += ", Tag: \(view.tag))\n"
```

The idea here is to give as much info as possible of the View being layouted

Reply:
It would probably feel weird to have it directly in `Layoutable` as it have nothing to do with layout. If the goal is to print as much info as possible, why not use the built-in `CustomDebugStringConvertible` protocol that already serve that purpose and print the tag if any and many more properties. We could add it as a requirement to `Layoutable`, what do you think? 
```swift
protocol Layoutable: AnyObject, Equatable, CustomDebugStringConvertible
```

Reply:
I didn't know this one, excellent idea.